### PR TITLE
Adding Box

### PR DESCRIPTION
**Pull request by AlfieBurns12345678910**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

File: data/en/bots/null.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+avatar: null
```
Comment (Andre601) [inline data/en/bots/null.md]: No avatar?

File: data/en/bots/null.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+cover: null
+custom_path: null
+description: This bot allows you to do magic
```
Comment (Andre601) [inline data/en/bots/null.md]: Common dude.... Put at least SOME effort into that.

Comment (AlfieBurns12345678910) [reply data/en/bots/null.md]: sorry, dont know how i got here, did this by accident.

File: data/en/bots/null.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+github: null
+images: []
+link: >-
+  https://discordapp.com/api/oauth2/authorize?response_type=code&client_id=157730590492196864&scope=identify%20guilds.join&state=15773059ghq9183habn&redirect_uri=https%3A%2F%2Fnicememe.website
```
Comment (Andre601) [inline data/en/bots/null.md]: wtf is that for an OAuth link?!

File: data/en/bots/null.md (hunk 2 above)
Comment (AlfieBurns12345678910) [reply data/en/bots/null.md]: ignore this post

**Comment by AlfieBurns12345678910** [comment]
bro, i did this by accident, i was trying to do something else, not sure how i ended up here.

**Comment by Andre601** [comment]
Then close it ffs. You have the power to do that

**Pull request closed without merging**